BiweeklyMeetings/2022-06-28.md

```diff
@@ -0,0 +1,442 @@
+
+**Robert** -: So I would say that we first try to create that channel to see how it works. And then we adjust the balance in both of those conversations to be sure that the less relevant stuff is confined to Discord and the most relevant stuff is recorded on GitHub. It will take practice, but I'm sure it can work. I do feel pretty strongly about that. And I'm willing to monitor it as well as do some of the digesting that Matthias just did, which I had intended also to do on my own at some point.
+
+**M. Àngels** -:  And can I suggest to ask each participant in this casual conversation, one, not an opinion, but a more reflected position on this to ask to add it to GitHub themselves and not do the summary, the editors.
+
+**Matthias** -: I think this can work if we discuss precise question on a Discord and then our goals, if we have precise goals, and then we bring back the results, the outcome of that discussion on the GitHub. For instance, one thing I would like to see people agree on is what would it means to be decentralized for Cardano, for this whole thing here. If people can agree on a definition of what decentralization is and would entail with something that is quantifiable and have that discussions because from the past discussion, it's not yet agreed between participant what it means.
+
+**Matthias** -: But once this is agreed that at least gives one criteria upon which we can start judging the CIP and the existing solution. And that will be one question. Then we move on to a different question or a different goal. And we try to structure the conversation on Discord around those goals. Anything that deviate from those goals, well, it should be self-disciplined. Instead of disciplinatory, not do it basically. So try to keep an eye on each other, but just say, no, this is not a topic. So the current topic and discussion we try to solve is this particular thing. So we break that big problem into smaller chunks and organize just discussions on one particular action on Discord up until we have done or tackle all of them.
+
+**M. Àngels** -:  We can give you a try. I mean, it's going to be a trial and error and we will need, as Robert said to learn from it. So let's give it a try if you agree.
+
+**Robert** -: Yes, it will be easier to do by setting up the discussion on that channel because the next time a contended issue comes along, it's going to blur the discussion. People may drop out or pay less focused attention. The framing questions that we might want to ask may get forgotten or push back farther in the discussions. I think that the only reason we've been able to focus on this particular issue is there aren't currently anyone like it that have attracted the same attention from the community. So we should move it off to a separate thread while we can do so easily before it gets mixed with other topics. So I would vote for creating that channel. Yes. And then to do its own.
```

Comment:
```suggestion
**Robert** -: Yes, it will be easier to do by setting up the discussion on that channel because the next time a contended issue comes along, it's going to blur the discussion. People may drop out or pay less focused attention. The framing questions that we might want to ask may get forgotten or push back farther in the discussions. I think that the only reason we've been able to focus on this particular issue is there aren't currently any ones like it that have attracted the same attention from the community. So we should move it off to a separate thread while we can do so easily before it gets mixed with other topics. So I would vote for creating that channel. Yes.
```
Don't remember what I said for the last phrase, but it wasn't that... 🤪